Add docs for extensions (#71)
vshn/appcat-user-docs: docs/modules/ROOT/pages/vshn-managed/postgresql/extensions.adoc

> = Add extensions to the instance

Reviewer: ```suggestion
= PostgreSQL Extensions
```

As the title appears on the navigation, I'd pick a shorter title here.

> PostgreSQL supports many different extenions, such as PostGIS and TimescaleDB.

Reviewer: ```suggestion
PostgreSQL supports many different extensions, such as https://postgis.net/[PostGIS^] and https://www.timescale.com/[TimescaleDB^].
```

> PostgreSQL supports many different extensions, such as https://postgis.net/[PostGIS^] and https://www.timescale.com/[TimescaleDB^].

Reviewer: ```suggestion
PostgreSQL supports many different extensions, such as https://postgis.net/[PostGIS^] or https://www.timescale.com/[TimescaleDB^].
```

These extensions can be used in PostgreSQL by VSHN as well.
For a list of all available extensions please https://stackgres.io/doc/latest/intro/extensions/[consult this page].

> == Enable extenions

Reviewer: ```suggestion
== Enable PostgreSQL Extensions
```

> For a list of all available extensions please https://stackgres.io/doc/latest/intro/extensions/[consult this page].

Reviewer: ```suggestion
For a list of all available extensions please https://stackgres.io/doc/latest/intro/extensions/[consult this page^].
```

Let's open external links in a new tab

> To enable an extension add the following to your instance:

Reviewer: ```suggestion
To enable an extension, add the following configuration to your instance:
```

[source,yaml]
----
apiVersion: vshn.appcat.vshn.io/v1
kind: VSHNPostgreSQL
metadata:
  name: pgsql-app1-prod
spec:
  parameters:
    service:
      extensions:

>         - name: ltree
> ----
> 

Reviewer: ```suggestion
        - name: ltree <1>
----

<1> Name of the extension as found on the "Name" column at https://stackgres.io/doc/latest/intro/extensions/[the official listing^]. Add more extensions to the list as needed.
```

> After applying the change AppCat will ensure that the extension is available after a short while.

Reviewer: ```suggestion
After applying the change, AppCat will ensure that the extension is available after a short while.
```

WARNING: Some extension like `timescaldb` or `postgis` will require a restart of the instance to be available.

> Depending on your https://docs.appcat.ch/vshn-managed/postgresql/update-strategy.html[Instance Update Strategy] it's possible that the instance will restart immediately.

Reviewer: ```suggestion
Depending on your https://docs.appcat.ch/vshn-managed/postgresql/update-strategy.html[Instance Update Strategy], it's possible that the instance will restart immediately.
```
